src/check/__tests__/check.test.js

```diff
@@ -80,4 +80,38 @@ describe("one-version: check", () => {
       });
     }).toThrow(FAILED_CHECK_ERROR);
   });
+
+  it("includes the manifest at -f in the check", () => {
+    const mockDetectPackageManager = jest.fn();
+    mockDetectPackageManager.mockReturnValue(packageManager);
+
+    const mockGetPackageDeps = jest.fn().mockReturnValue({});
+    const mockIsValidWorkspace = jest.fn().mockReturnValue(true);
+    const mockGetWorkspaces = jest.fn().mockReturnValue([]);
+
+    check({
+      file: "some-existent-path",
+      getPackageManager: mockDetectPackageManager,
+      getConfig: mockGetConfig,
+      getDependencies: mockGetPackageDeps,
+      getWorkspaces: mockGetWorkspaces,
+      validateWorkspace: mockIsValidWorkspace,
+    });
+
+    expect(mockGetPackageDeps).toHaveBeenCalledWith("some-existent-path");
```

Comment:
Let's make sure `mockPackageDeps` is also being called with any other workspaces returned from `getWorkspaces`